### PR TITLE
Scoreboard and Karma Toggles

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Tired of seeing the karma reminder? Sick of being reminded how awful the crew is? Tired of clicking "X" on two windows every round? Do you not give a flying fart in space about karma? Perhaps you just want to get to robusting the living shit out of your fellow spaceman as soon as possible without having any sort of obnoxious popup. Well...this is the PR for you then!

- Adds in toggles for end of round scoreboard and karma reminder

Simply put, if you don't want to see the round end scoreboard or the end of round karma reminder, then you can simply toggle them off under the preferences tab and never worry about seeing them again (if/until you re-enable them, of course).

### CODE REVIEW

**Comment by Iamgoofball** [comment]
These are enabled to 1 by default, right?

**Comment by Fox-McCloud** [comment]
Yeah, by default, the scoreboard and the karma indicators WILL show up.
